crates/eips/src/eip7910.rs

```diff
@@ -0,0 +1,263 @@
+//! Implementation of [`EIP-7910`](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-7910.md).
+
+use crate::{eip2935, eip4788, eip6110, eip7002, eip7251, eip7840::BlobParams};
+use alloc::{borrow::ToOwned, collections::BTreeMap, string::String};
+use alloy_primitives::{Address, Bytes, U64};
+use core::{fmt, str};
+
+/// Response type for `eth_config`
+#[derive(Clone, Debug, PartialEq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(feature = "serde", serde(rename_all = "camelCase"))]
+pub struct EthConfig {
+    /// Fork configuration of the current active fork.
+    pub current: EthForkConfig,
+    /// Fork configuration of the next scheduled fork.
+    pub next: Option<EthForkConfig>,
+    /// Fork configuration of the last fork (before current).
+    pub last: Option<EthForkConfig>,
+}
+
+/// The fork configuration object as defined by [`EIP-7910`](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-7910.md).
+#[derive(Clone, Debug, PartialEq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(feature = "serde", serde(rename_all = "camelCase"))]
+pub struct EthForkConfig {
+    /// The fork activation timestamp, represented as a JSON number in Unix epoch seconds (UTC).
+    /// For the "current" configuration, this reflects the actual activation time; for "next," it
+    /// is the scheduled time. Activation time is required. If a fork is activated at genesis
+    /// the value `0` is used. If the fork is not scheduled to be activated or its activation time
+    /// is unknown it should not be in the rpc results.
+    pub activation_time: u64,
```

Comment:
~~is this correct that we have a mix of numbers and hex here?~~

it is actually...